data/core/faqs.md

```diff
@@ -911,14 +911,14 @@
     - utter_ask_explain_nlucorex
 * deny
     - utter_direct_to_step2
-* ask_faq_ee OR ask_faq_languages OR ask_faq_tutorialcore OR ask_faq_tutorialnlu OR ask_faq_opensource OR ask_faq_voice OR ask_faq_slots OR ask_faq_channels OR ask_faq_differencecorenlu OR ask_faq_python_version OR ask_faq_community_size OR ask_faq_what_is_forum OR ask_faq_tutorials OR ask_faq_differencerasarasax OR ask_faq_rasax
+* ask_faq_ee OR ask_faq_languages OR ask_faq_tutorialcore OR ask_faq_tutorialnlu OR ask_faq_opensource OR ask_faq_voice OR ask_faq_slots OR ask_faq_channels OR ask_faq_differencecorenlu OR ask_faq_differencerasarasax OR ask_faq_python_version OR ask_faq_community_size OR ask_faq_what_is_forum OR ask_faq_tutorials OR ask_faq_rasax
     - action_faqs
     - utter_direct_to_step2
 
 ## not new to rasa + not interested in products
 * greet
     - action_greet_user
-* ask_faq_ee OR ask_faq_languages OR ask_faq_tutorialcore OR ask_faq_tutorialnlu OR ask_faq_opensource OR ask_faq_voice OR ask_faq_slots OR ask_faq_channels OR ask_faq_differencecorenlu OR ask_faq_python_version OR ask_faq_community_size OR ask_faq_what_is_forum OR ask_faq_tutorials OR ask_faq_differencerasarasax OR ask_faq_rasax
+* ask_faq_ee OR ask_faq_languages OR ask_faq_tutorialcore OR ask_faq_tutorialnlu OR ask_faq_opensource OR ask_faq_voice OR ask_faq_slots OR ask_faq_channels OR ask_faq_differencecorenlu OR ask_faq_differencerasarasax OR ask_faq_python_version OR ask_faq_community_size OR ask_faq_what_is_forum OR ask_faq_tutorials OR ask_faq_rasax
```

Comment:
I know these are functionally the same, but might be good to just copy over the old file so that there is no unnecessary diffs

Reply:
I did a git cherry-pick to put it back. The branch shows it is the same but under the PR it still shows the difference.

Reply:
i think you would then have to force-push the branch afterward right?